annotation_pipeline/molecular_db.py

```diff
@@ -72,13 +72,13 @@ def _store(segm_i):
 
     pw = pywren.ibm_cf_executor(config=config, runtime_memory=2048)
     futures = pw.map(generate_formulas, adducts)
-    segments_n = set().union(*pw.get_result(futures))
+    segments_n = list(set().union(*pw.get_result(futures)))
```

Comment:
with recent PyWren version, we can also use set `iterdata`s

Reply:
Iterables aren't supported in PyWren 1.1.1. I'm unable to use PyWren 1.2.0 because of this issue: https://github.com/pywren/pywren-ibm-cloud/issues/237 .